masschallenge/impact-api
# [AC-4959] Add support for accepting the X-HTTP-Method-Override Header

**To Test**
Browser Window
- Login to the API (http://localhost:8000/api/impact/) and create a new Client ID.
  - Select Client Type "Public".
  - Select Authorization grant type "Resource owner password-based"
  - Copy the Client ID. 
- Go to http://localhost:8000/api/v1/user/45679/.  Observe the name and email fields.

Postman
- Open up Postman
- Go to Import > Import From Link
  - Use https://www.getpostman.com/collections/e9040bab8bb0b9691a83
- Under localhost > v1, you should see a call called "v1 user patch via post".
- In the body, make changes to the existing fields.
- Click send.
  - You should see a response of 204.

Browser Window 
- Refresh the page.
  - You should see your changes in the fields you altered.
